### PR TITLE
PRIME-2386 User Preferred Names in Emails

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-dotnet-webapi/Services/EmailService.cs [hunk 1]
```diff
@@ -64,8 +64,8 @@ public async Task SendProvisionerLinkAsync(string[] emails, EnrolmentCertificate
                 .Where(e => e.Id == token.EnrolleeId)
                 .Select(e => new
                 {
-                    e.FirstName,
-                    e.LastName,
+                    FirstName = e.PreferredFirstName ?? e.FirstName,
+                    LastName = e.PreferredLastName ?? e.LastName,
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/EmailService.cs:68]: Should this be refactored into common functions or property?

Comment (bergomi02) [reply prime-dotnet-webapi/Services/EmailService.cs:68]: Sure, but don't know sure what the property name should be. Any idea?

Comment (neophyte57) [reply prime-dotnet-webapi/Services/EmailService.cs:68]: What do you think of `Current`?

Comment (bergomi02) [reply prime-dotnet-webapi/Services/EmailService.cs]: The code is updated.

**Review by neophyte57 — APPROVED** [review]
LGTM!

**Pull request merged**